src/visualization/plot_recipes.jl

```diff
@@ -327,7 +327,7 @@ function PlotData1D(u, semi;
 
   variable_names = SVector(varnames(solution_variables, equations))
 
-  return PlotData1D(vec(x), vec(u), variable_names, vcat(x[1, 1, :], x[1, end, end]))
+  return PlotData1D(vec(x), u, variable_names, vcat(x[1, 1, :], x[1, end, end]))
```

Comment:
I want to let u as it is here, because casting it into a vector makes it harder to work with when you have multiple variables.